lib/arInstall.class.php

```diff
@@ -0,0 +1,421 @@
+            as $item
+        ) {
+            $premisAccessRightValues[$item->slug] = [
+                'allow_master' => 1,
+                'allow_reference' => 1,
+                'allow_thumb' => 1,
+                'conditional_master' => 0,
+                'conditional_reference' => 1,
+                'conditional_thumb' => 1,
+                'disallow_master' => 0,
+                'disallow_reference' => 0,
+                'disallow_thumb' => 0,
+            ];
+        }
+        $setting = new QubitSetting();
+        $setting->name = 'premisAccessRightValues';
+        $setting->sourceCulture = sfConfig::get('sf_default_culture');
+        $setting->setValue(serialize(
+            $premisAccessRightValues),
+            ['sourceCulture' => true]
+        );
+        $setting->save();
+
+        $i18n = sfContext::getInstance()->i18n;
+        $accessDisallowWarning = $i18n->__(
+            'Access to this record is restricted because it contains personal'
+            .' or confidential information. Please contact the Reference'
+            .' Archivist for more information on accessing this record.'
+        );
+        $accessConditionalWarning = $i18n->__(
+            'This record has not yet been reviewed for personal or'
+            .' confidential information. Please contact the Reference'
+            .' Archivist to request access and initiate an access review.'
+        );
+
+        foreach (
+            QubitTaxonomy::getTermsById(QubitTaxonomy::RIGHT_BASIS_ID)
+            as $item
+        ) {
+            $setting = new QubitSetting();
+            $setting->name = "{$item->slug}_disallow";
+            $setting->scope = 'access_statement';
+            $setting->setValue($accessDisallowWarning, ['culture' => 'en']);
+
+            foreach (
+                QubitI18N::getTranslations($accessDisallowWarning)
+                as $langCode => $message
+            ) {
+                $setting->setValue($message, ['culture' => $langCode]);
+            }
+
+            $setting->save();
+
+            $setting = new QubitSetting();
+            $setting->name = "{$item->slug}_conditional";
+            $setting->scope = 'access_statement';
+            $setting->setValue($accessConditionalWarning, ['culture' => 'en']);
+
+            foreach (
+                QubitI18N::getTranslations($accessConditionalWarning)
+                as $langCode => $message
+            ) {
+                $setting->setValue($message, ['culture' => $langCode]);
+            }
+
+            $setting->save();
+        }
+    }
+
+    public static function populateSearchIndex()
+    {
+        sfConfig::add(QubitSetting::getSettingsArray());
+        QubitSearch::enable();
+        QubitSearch::getInstance()->populate();
+    }
+
+    public static function createSetting($name, $value)
+    {
+        $setting = new QubitSetting();
+        $setting->name = $name;
+        $setting->value = $value;
+        $setting->save();
```

Comment:
Nitpick - insert newline before 419.

Reply:
Why?

Reply:
Please disregard! I believe I have misinterpreted the PHP-CS-Fixer rule.